src/net/relay.rs

```diff
@@ -2189,6 +2210,33 @@ mod test {
         push_message(peer, dest, relay_hints, msg)
     }
 
+    fn broadcast_block(
+        peer: &mut TestPeer,
+        relay_hints: Vec<RelayData>,
+        consensus_hash: ConsensusHash,
+        block: StacksBlock,
+    ) -> bool {
+        test_debug!(
+            "{:?}: Broadcast block {}/{}",
+            peer.to_neighbor().addr,
+            &consensus_hash,
+            block.block_hash(),
+        );
+
+        let sn = SortitionDB::get_block_snapshot_consensus(
```

Comment:
Why invoke `get_block_snapshot_consensus` when this function already is given a consensus_hash?

Reply:
To force the test to panic if the consensus hash doesn't exist.